src/lib/LaunchDarklyClient/index.ts

```diff
@@ -1,16 +1,23 @@
 import hash from 'hash.js';
 import * as LDClient from 'ldclient-js';
 
+interface IUser extends LDClient.LDUser {
+  authId: string
+}
+interface IOptions extends LDClient.LDOptions {
+  baseTimeout: number
+}
+
 class LDApi {
 
   private ldClient: LDClient.LDClient;
   private readonly env: String;
 
-  constructor(env) {
+  constructor(env: string) {
     this.env = env;
   }
 
-  init(user, envClientKey, options, timeout, logUpdates = false) {
+  init(user: IUser, envClientKey: string, options: IOptions, timeout: number, logUpdates: boolean = false) {
```

Comment:
Is there a reason for doing these inline (repeatedly) versus once at the top of the document? I would suggest cleaning that up.

Reply:
How can we specify multiple function parameters' type once at the top of the document?